lib/rules/web/admin_console/4.8/dashboards.xyaml

```diff
@@ -687,6 +687,7 @@ click_filesystem_chart:
     selector:
       <<: *filesystem_chart
     op: click
+check_charts_on_deployment_page: {}
```

Comment:
Some checking may need on OCP4.8 for this function, let's check if it is a bug on console first

Reply:
As OCP4.6-4.8 version is already in maintenance mode, only security fixes should be fielded. So no needs update for this function anymore
Add a note for future tracking: Bug 2073922